gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/StudentGradeSummaryGradesPanel.java

```diff
@@ -131,7 +131,7 @@ public void onBeforeRender() {
 				if (!catItems.isEmpty()) {
 					Long catId = catItems.get(0).getCategoryId();
 					if (catId != null) {
-						businessService.getCategoryScoreForStudent(catId, userId)
+						businessService.getCategoryScoreForStudent(catId, userId, false) // Dont include non-released items in the category calc
```

Comment:
I think this panel is used for both the student and instructor views. Always passing false here might be a problem on the student grade summary panel. It is probably appropriate for the Student Review Mode view but not the Grade Summary view which shows all gradebook items.

Reply:
I tested both instructor and student view .... more testing would def be appreciated. I'm also looking for general feedback on the correct approach here. I think there is an email thread on sakai-user about expectations

Reply:
I took it for a spin locally and it does all seem fine to me. The Grade Summary view shows the item and factors it into the category calculation, while the Student Review Mode hides the items and does not include it in the calc. Although we're not showing the student their true category score, it is the correct score from their point of view.